gateway/testutil.go

```diff
@@ -1306,6 +1305,13 @@ func GetTLSClient(cert *tls.Certificate, caCert []byte) *http.Client {
 		tlsConfig.InsecureSkipVerify = true
 	}
 
+	return tlsConfig
+}
+
+func GetTLSClient(cert *tls.Certificate, caCert []byte) *http.Client {
```

Comment:
I'm asuming these can be unexposed for now ; should move out of scope into /internal/tls or something. The current discussion is that we mirror the x/ stdlib structure, e.g. `/internal/crypto/tls`. Not a blocker.